org.eclipse.lsp4e.debug/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,8 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Debug Adapter client for Eclipse IDE (Incubation)
 Bundle-SymbolicName: org.eclipse.lsp4e.debug;singleton:=true
-Bundle-Version: 0.15.11.qualifier
+Bundle-Vendor: Eclipse.org
```

Comment:
https://www.eclipse.org/projects/handbook/#code states that

> Provider information is set to the project’s formal name: e.g. the Bundle-Vendor entry set to "Eclipse Foo" in OSGi Bundles;

So it should be
```suggestion
Bundle-Vendor: Eclipse LSP4E
```